ev3dev2/led.py

```diff
@@ -81,15 +113,14 @@ class Led(Device):
     def __init__(self,
                  name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, name_exact=False,
                  desc=None, **kwargs):
+        self.desc = desc
```

Comment:
This is used in ``__str__()`` so we need to set it before calling the super __init__